### PR TITLE
[Serverless][8.18] EQL Sequence alert suppression 

### PR DESCRIPTION
**Pull request by nastasha-solomon**
Fixes https://github.com/elastic/security-docs/issues/5886

### ESS
- [Alert suppression](https://security-docs_bk_6291.docs-preview.app.elstc.co/guide/en/security/master/alert-suppression.html) -  Made the following changes:
   - Removed the second bullet in the requirements note at the start of the page since alert suppression for EQL rules will be GA in 8.18.
   - Updated the note after step 1 that explains how suppression fields with multiple values are handled. Added a third bullet for event correlation rules using sequence queries. 
- [Create a detection rule | Create an event correlation rule](https://security-docs_bk_6291.docs-preview.app.elstc.co/guide/en/security/master/rules-ui-create.html#create-eql-rule) - Removes the tech preview label from step  6, which lets users know that they can suppress alerts from this rule type.

### Serverless

**NOTE:** Suppression for EQL rules in Serverless will go GA around the same time that 8.18 GAs. I'll open a separate PR to update the Serverless [docs](https://www.elastic.co/guide/en/serverless/current/security-rules-create.html#create-eql-rule) once that date approaches. 

- [Alert suppression](https://security-docs_bk_6291.docs-preview.app.elstc.co/guide/en/serverless/main/security-alert-suppression.html) - Same changes as described above

### CODE REVIEW

**Comment by natasha-moore-elastic** [comment]
Per [this comment](https://github.com/elastic/security-docs/pull/6224/files#r1883951350), this feature didn't make it into 8.17.

File: docs/detections/alert-suppression.asciidoc [hunk 1]
```diff
@@ -45,7 +45,9 @@ If you specify a field with multiple values, alerts with that field are handled
...
 * **Indicator match, event correlation (non-sequence queries only), new terms, {esql}, or {ml} rules:** Alerts with the specified field name and identical array values are grouped together. For example, if you suppress alerts by `destination.ip` of `[127.0.0.1, 127.0.0.2, 127.0.0.3]`, alerts with the entire array are grouped and only one alert is created for the group.
-
+* **Event correlation (sequence queries only) rules:** Be aware of the following:
+** If the specified field contains an array of values, suppression only happens if the field's values are an exact match and in the same order. For example, if you specify the field `myips` and one sequence alert has [1.1.1.1, 0.0.0.0] and another sequence alert has [1.1.1.1, 192.168.0.1], neither of those alerts will be suppressed, despite sharing an array element.
+** Suppression is only supported for sequence alerts, not <<security-building-block-rules,building block alerts>>.
```
Comment (nastasha-solomon) [inline docs/detections/alert-suppression.asciidoc:50]:
```suggestion
** Suppression is only supported for sequence alerts, not <<building-block-rules,building block alerts>>.
```

**Review by dhurley14 — COMMENTED** [review]
Just one clarification. Looks great!

File: docs/detections/building-block-rule.asciidoc [hunk 2]
```diff
@@ -18,6 +18,8 @@ To create a rule that searches alert indices, select *Index Patterns* as the rul
 [role="screenshot"]
 image::images/alert-indices-ui.png[]
 
+NOTE: Suppressing building block alerts is not supported, but you can suppress alerts generated by event correlation rules with sequence queries. Refer to <<alert-suppression>> to learn more.
```
[the remark this answers is not in the document]
Comment (nastasha-solomon) [reply docs/detections/building-block-rule.asciidoc:21]:
```suggestion
```

File: docs/serverless/rules/building-block-rule.asciidoc [hunk 3]
```diff
@@ -22,6 +22,8 @@ To create a rule that searches alert indices, select **Index Patterns** as the r
 [role="screenshot"]
 image::images/building-block-rule/-detections-alert-indices-ui.png[]
 
+NOTE: Suppressing building block alerts is not supported, but you can suppress alerts generated by event correlation rules with sequence queries. Refer to <<security-alert-suppression>> to learn more.
```
Comment (nastasha-solomon) [inline docs/serverless/rules/building-block-rule.asciidoc:25]:
```suggestion
```

**Review by dhurley14 — APPROVED** [review]

**Review by jmikell821 — DISMISSED** [review]

**Review by benironside — APPROVED** [review]

**Pull request merged**